tensorflow_lite_support/web/tflite_model_runner/demo/dist/worker.js

```diff
@@ -0,0 +1,231 @@
+        postMessage(loadFinishedMs);
+        break;
+
+      case "infer":
+
+        //////////////////////////////////////////////////////////////////////////////
+        // Get input and output info.
+
+        const inputs = callAndDelete(modelRunner.GetInputs(), (results) =>
+          convertCppVectorToArray(results)
+        );
+        const input = inputs[0];
+        const outputs = callAndDelete(modelRunner.GetOutputs(), (results) =>
+          convertCppVectorToArray(results)
+        );
+        const output = outputs[0];
+
+        //////////////////////////////////////////////////////////////////////////////
+        // Set input tensor data from the image (224 x 224 x 3).
+
+        const { vals, width, height } = message.data[1]
+        if (!vals) return;
+        const inputBuffer = input.data();
+        const inputData = new Float32Array(inputBuffer.length);
+        let pixelIndex = 0;
+        for (let i = 0; i < width; i++) {
+          for (let j = 0; j < height; j++) {
+            const valStartIndex = pixelIndex * 4;
+            const inputIndex = pixelIndex * 3;
+            inputData[inputIndex] = (vals[valStartIndex] - 127.5) / 127.5;
+            inputData[inputIndex + 1] =
+              (vals[valStartIndex + 1] - 127.5) / 127.5;
+            inputData[inputIndex + 2] =
+              (vals[valStartIndex + 2] - 127.5) / 127.5;
+            pixelIndex += 1;
+          }
+        }
+        inputBuffer.set(inputData);
+
+        //////////////////////////////////////////////////////////////////////////////
+        // Infer, get output tensor, and sort by logit values in reverse.
+
+
+        const numRuns = message.data[2]
+        const inferTimes = [];
+        for (let i = 0; i < numRuns; i++) {
+          const start = performance.now();
+          const success = modelRunner.Infer();
+          const inferTime = (performance.now() - start).toFixed(2);
+          if (!success) return;
+          console.log(`Infer time ${i + 1}: ${inferTime} ms`);
+          inferTimes.push(Number(inferTime));
+        }
+
+        const result = Array.from(output.data());
+        result.shift(); // Remove the first logit which is the background noise.
+        const sortedResult = result
+          .map((logit, i) => {
+            return { i, logit };
+          })
+          .sort((a, b) => b.logit - a.logit);
+        postMessage({sortedResult,inferTimes})
+      default:
+        break;
+    }
+  }
+};
+
+////////////////////////////////////////////////////////////////////////////////
+// Helper functions.
+
+function getMedianValue(array) {
+  array = array.sort((a, b) => a - b);
+  const medianValue =
+    array.length % 2 !== 0
+      ? array[Math.floor(array.length / 2)]
+      : (array[array.length / 2 - 1] + array[array.length / 2]) / 2;
+  return medianValue.toFixed(2);
+}
+
+let fromPixels2DContext;
+
+/** Gets the raw pixel data from the given HTML element. */
+function fromPixels(pixels) {

```

Comment:
This function should be duplicated with the same one in `script.js`, it's not needed in `worker.js`.